libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -0,0 +1,890 @@
+    }
+
+    static String vcExperianFraudScoreOne() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder()
+                        .address(List.of(ADDRESS_3))
+                        .passport(null)
+                        .build();
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .evidence(FRAUD_EVIDENCE_SCORE_ONE)
+                        .credentialSubject(credentialSubject)
+                        .build(),
+                "urn:uuid:7fadacac-0d61-4786-aca3-8ef7934cb092",
+                "https://review-f.staging.account.gov.uk",
+                Instant.ofEpochSecond(1704290386));
+    }
+
+    static String vcExperianFraudScoreTwo() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder()
+                        .name(
+                                List.of(
+                                        Map.of(
+                                                VC_NAME_PARTS,
+                                                List.of(new NameParts("Chris", VC_GIVEN_NAME)))))
+                        .address(List.of(Map.of("type", "PostalAddress", "postalCode", "LE12 9BN")))
+                        .birthDate(List.of(new BirthDate("1984-09-28")))
+                        .passport(null)
+                        .build();
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .evidence(FRAUD_EVIDENCE_CRI_STUB_CHECK)
+                        .credentialSubject(credentialSubject)
+                        .build(),
+                "user-id",
+                "https://review-f.staging.account.gov.uk",
+                Instant.ofEpochSecond(1652953380));
+    }
+
+    static String vcExperianFraudMissingName() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder()
+                        .name(Collections.emptyList())
+                        .address(List.of(ADDRESS_3))
+                        .passport(null)
+                        .build();
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .evidence(FRAUD_EVIDENCE_CRI_STUB_CHECK)
+                        .credentialSubject(credentialSubject)
+                        .build(),
+                "urn:uuid:7fadacac-0d61-4786-aca3-8ef7934cb092",
+                "https://review-f.staging.account.gov.uk",
+                Instant.ofEpochSecond(1704290386));
+    }
+
+    static String vcDrivingPermit() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder()
+                        .address(List.of(ADDRESS_4))
+                        .drivingPermit(List.of(DRIVING_PERMIT))
+                        .passport(null)
```

Comment:
I'm wondering if defining passport with a default value in `TestCredentialSubject` might not be the best thing to do now. Forces you to set all these nulls, and it could easily be missed when creating new fixtures.